strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BrowseController.java

```diff
@@ -145,26 +151,17 @@
             
             String matchedPattern = (String) request.getAttribute(HandlerMapping.BEST_MATCHING_PATTERN_ATTRIBUTE);
             String subPath = new AntPathMatcher().extractPathWithinPattern(matchedPattern, request.getRequestURI());
```

Comment:
take a look how `path` variable implemented in `MavenArtifactController`. You can try to use `{storageId}/{repositoryId}/{path:.+}` to expose it. I think this can suit well here.